lib/config.service.ts

```diff
@@ -9,12 +11,18 @@ import { NoInferType } from './types';
 
 @Injectable()
 export class ConfigService<K = Record<string, any>> {
```

Comment:
Imo, there's no reason for `K` type argument here. We can safely remove it

Reply:
Maybe I should make this change on a new PR to keep this feature from being blocked until a new major version?
What do you think @kamilmysliwiec ?

Reply:
Ugh, apologies! I completely forgot that we use this type argument elsewhere already, my mistake. Ignore this suggestion then!